demisto_sdk/commands/common/tools.py

```diff
@@ -1355,27 +1378,19 @@ def get_ignore_pack_skipped_tests(
     file_name_to_ignore_dict: Dict[str, List[str]] = {}
     test_playbooks = id_set.get("TestPlaybooks", {})
 
-    pack_ignore_path = get_pack_ignore_file_path(pack_name)
-    if pack_name in modified_packs:
-        if os.path.isfile(pack_ignore_path):
-            try:
-                # read pack_ignore using ConfigParser
-                config = ConfigParser(allow_no_value=True)
-                config.read(pack_ignore_path)
-
-                # go over every file in the config
-                for section in config.sections():
-                    if section.startswith("file:"):
-                        # given section is of type file
-                        file_name: str = section[5:]
-                        for key in config[section]:
-                            if key == "ignore":
-                                # group ignore codes to a list
-                                file_name_to_ignore_dict[file_name] = str(
-                                    config[section][key]
-                                ).split(",")
-            except MissingSectionHeaderError:
-                pass
+    # pack_ignore_path = get_pack_ignore_file_path(pack_name)
+    if pack_name in modified_packs and (config := get_pack_ignore_content(pack_name)):
+        # go over every file in the config
+        for section in config.sections():
+            if section.startswith("file:"):
+                # given section is of type file
+                file_name: str = section[5:]
+                for key in config[section]:
+                    if key == "ignore":
+                        # group ignore codes to a list
+                        file_name_to_ignore_dict[file_name] = str(
```

Comment:
theres a lot of nesting here and it isnt clear what we're doing. Lets split into functions with names that make it clearer

Reply:
perhaps using `more_itertools`, or some Sourcery

Reply:
this code is old, wasn't actually changed by me, so after speaking with @JudahSchwartz ill disregard it